doc/content/getting-started/migrating/migrating-from-v2.md

```diff
@@ -25,11 +25,24 @@ When migrating devices from the public {{< ttnv2 >}} to {{< tts >}} Cloud, you m
 
 When migrating from a private {{% ttnv2 %}}, devices that are outside of the DevAddr address block supported by {{% tts %}} Cloud will have to rejoin the network, otherwise {{% tts %}} will be unable to route their uplink and downlink traffic.
 
-## Configure ttn-lw-migrate
+{{< warning >}} Migrating device sessions will not work on {{% tts %}} v3.10 or older versions. {{</ warning >}}
+
+## Pre-requisites
+
+1. User account in V2.
```

Comment:
User account in {{% ttnv2 %}}
User account in {{% tts %}} v3

Reply:
Done